tests/scripts/single_card/nightly/run_ttnn.sh

```diff
@@ -8,7 +8,11 @@ if [[ -z "$TT_METAL_HOME" ]]; then
 fi
 fail=0
 
-echo "Running ttnn nightly tests for GS only"
+echo "Running ttnn nightly tests"
+
+if [[ "$ARCH_NAME" == "wormhole_b0" ]]; then
+  export WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml
```

Comment:
why do you want to use this by default for WH?

Reply:
I think quite a few models depend on this being the case except for some specific N150 ones 🥴 

There's not really a way to change it via fixtures unless we add some more complex fixture logic. I think I could try making a separate call for N150 jobs to try without the env var?